src/main/java/io/github/externschool/planner/aop/LoggingAspect.java

```diff
@@ -0,0 +1,75 @@
+package io.github.externschool.planner.aop;
+
+
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import org.aspectj.lang.reflect.MethodSignature;
+
+
+import java.util.Arrays;
+
+@Aspect
+@Component
+public class LoggingAspect {
+    private Logger log = LoggerFactory.getLogger(getClass());
+
+    @Pointcut("within(io.github.externschool.planner.controller..*) || within(io.github.externschool.planner.service..*) ")
+    private void allMethods() {
+
+    }
+
+        @Before("allMethods()")
+    public void log(JoinPoint joinPoint) {
+        Object[] arguments = joinPoint.getArgs();
+        String className = joinPoint.getSignature().getDeclaringTypeName();
+        String methodName = joinPoint.getSignature().getName();
+        System.out.println("before" + className + "." + methodName + "() -----");
+
+        for (int i = 0; i < arguments.length; i++) {
+            System.out.println(arguments[i]);
+        }
+    }
+    @Around("allMethods()")
+    public Object logBusinessMethods(ProceedingJoinPoint call) throws Throwable {
+        if (!log.isDebugEnabled()) {
+            return call.proceed();
+        } else {
+
+            Object[] args = call.getArgs();
+            String message = call.toShortString();
+            log.info("{} called with args '{}'!", message, Arrays.deepToString(args));
+            Object result = null;
+
+
```

Comment:
The second blank line is not necessary here. Single line could be used as a logic blocks delimiter or if you want to emphasize something like a return statement for example. Extra lines make your code hard to read.